docs/source/user/running.rst

```diff
@@ -10,6 +10,9 @@ directories:
 .. image:: ../images/running-layout.png
    :align: center
    :class: jp-screenshot
+   :alt:
+
+<!---Alt text is intentionally left blank because the image content is described directly prior.--->
```

Comment:
```suggestion
.. only:: comment
   Alt text is intentionally left blank because the image content is described directly prior.
```

Currently it does gets displayed. I haven't tried the suggestion, but https://stackoverflow.com/questions/4783814/how-to-comment-a-string-in-restructured-text says it should work

![Screenshot from 2022-07-31 14-57-19](https://user-images.githubusercontent.com/5832902/182029813-3185cf86-9ed4-41b2-9657-208b782605cf.png)